docs/iidm/model/phaseTapChanger.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Phase tap changer
+layout: default
+---
+
+The `com.powsybl.iidm.network.PhaseTapChanger` interface is used to model phase tap changer devices on transformers.
+It can be added to both [two windings transformers](./twoWindingsTransformer.md) and [three windings transformers](./threeWindingsTransformer.md).
+
+# Characteristics
+
+| Attribute | Type | Unit | Required | Default value | Description |
+| --------- | ---- | ---- | -------- | ------------- | ----------- |
+| Low tap position | int | - | no | 0 | Position index of the tap changer's low tap |
+| Tap position | int | - | yes | - | Position index of current tap |
+| Regulating | boolean | - | no | false | True if ratio tap changer is regulating, false otherwise. State variable |
+| Regulation mode | enum | - | no | FIXED_TAP | Regulation mode of the phase tap changer. May be CURRENT_LIMITER, ACTIVE_POWER_CONTROL or FIXED_TAP |
+| Regulation value | double | kV | yes | - | The target voltage |
```

Comment:
The regulating value of the phase tap changer is not a voltage : it could be an active power value (for active power control mode) or a current (for current limiter mode)